src/test/java/com/ge/predix/uaa/token/lib/FastTokenServiceTest.java

```diff
@@ -1,303 +0,0 @@
-        assertNotNull(result);
-        assertEquals("cf", result.getOAuth2Request().getClientId());
-        assertEquals("marissa", result.getUserAuthentication().getName());
-        assertEquals("1adc931e-d65f-4357-b90d-dd4131b8749a",
-                ((RemoteUserAuthentication) result.getUserAuthentication()).getId());
-        assertNotNull(result.getOAuth2Request().getRequestParameters());
-        assertNull(result.getOAuth2Request().getRequestParameters().get(Claims.ISS));
-    }
-
-    @Test
-    public void testLoadAuthenticationLoadingCacheExpiry() throws Exception {
-        FastTokenServices fastTokenServices  = Mockito.spy(new FastTokenServices());
-        fastTokenServices.setTrustedIssuers(trustedIssuers());
-        fastTokenServices.setIssuerPublicKeyTTLMillis(1000L);
-        fastTokenServices.setRestTemplate(mockRestTemplate(false));
-        fastTokenServices.afterPropertiesSet();
-        String accessToken = this.testTokenUtil.mockAccessToken(60);
-        //Populate the Cache (Cache computed)
-        fastTokenServices.loadAuthentication(accessToken);
-        Mockito.verify(fastTokenServices,times(1)).computeSignatureVerifier(TOKEN_ISSUER_ID);
-        //Call to make sure that SignatureVerifier is (Cache retrieved)
-        fastTokenServices.loadAuthentication(accessToken);
-        Mockito.verify(fastTokenServices,times(1)).computeSignatureVerifier(TOKEN_ISSUER_ID);
-        //Ensure the Cache entry for issuer times out
-        Thread.sleep(1100L);
-        //Call that tries to populate the Cache again
-        fastTokenServices.loadAuthentication(accessToken);
-        //Validate compute call is made when the entry is not available in Cache (Cache expired and computed again)
-        Mockito.verify(fastTokenServices,times(2)).computeSignatureVerifier(TOKEN_ISSUER_ID);
-    }
-
-    @Test
-    public void testLoadAuthenticationForUpdatedIssuerTokenSigningKeyPositive() throws Exception {
-        FastTokenServices fastTokenServices  = new FastTokenServices();
-        fastTokenServices.setTrustedIssuers(trustedIssuers());
-        fastTokenServices.setIssuerPublicKeyTTLMillis(3000L);
-        fastTokenServices.setRestTemplate(mockRestTemplate(false));
-        fastTokenServices.afterPropertiesSet();
-        String accessToken = this.testTokenUtil.mockAccessToken(60, false);
-        OAuth2Authentication result = fastTokenServices.loadAuthentication(accessToken);
-        assertNotNull(result);
-
-        fastTokenServices.setRestTemplate(mockRestTemplate(true));
-        //Ensure the TokenKey for issuer times out
-        Thread.sleep(3100L);
-        accessToken = this.testTokenUtil.mockAccessToken(60, true);
-        result = fastTokenServices.loadAuthentication(accessToken);
-        assertNotNull(result);
-    }
-
-    @Test(expectedExceptions = InvalidSignatureException.class)
-    public void testLoadAuthenticationForUpdatedIssuerTokenSigningKeyNegative() throws Exception {
```

Comment:
can we add missing tests from this class to src/test/java/com/ge/predix/uaa/token/lib/FastTokenServicesTest.java ?

Reply:
I don't think this is important, so I skipped this test.